app/services/database-service.js

```diff
@@ -16,9 +16,9 @@ module.exports = (() => {
   fs.readdirSync(modelPath)
     .filter((file) => {
       return (
-        file.indexOf('.') !== 0 &&
+        file.startsWith('.') &&
```

Comment:
I think this should be doesn't start with, I think that's what the previous equality is looking for the file to not start with a '.'